lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,19 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+// eslint-disable-next-line sort-keys
+  'airbnb-prop-types-to-prop-types-tools': { // eslint-disable-line sort-keys
```

Comment:
```suggestion
  'airbnb-prop-types-to-prop-types-tools': {
```

Whoops, we don't need this then. :see_no_evil: